app/renderer/components/Inspector/HeaderButtons.js

```diff
@@ -15,13 +16,17 @@ import {
   SearchOutlined,
   CloseOutlined,
   AppstoreOutlined,
-  GlobalOutlined
+  GlobalOutlined,
+  InfoCircleOutlined,
+  ExclamationCircleOutlined
 } from '@ant-design/icons';
 
+const HYBRID_MODE_DOCS_URL = 'https://github.com/appium/appium/blob/1.x/docs/en/writing-running-appium/web/hybrid.md';
```

Comment:
@jlipps Might this doc be outdated?

Reply:
If we need the link, maybe https://appium.github.io/appium.io/docs/en/writing-running-appium/web/hybrid/ will be better. but afaik the documentation is outdated. (ideally, each driver that supports hybrid need something)

Reply:
I've now changed the link to the appium.github.io version, I agree that it looks better.

Reply:
we could link to https://appium.io/docs/en/2.0/guides/context/, and then link from this page to driver-specific context docs as necessary

Reply:
I'm more than happy to update the docs links (including others elsewhere in the code), but I would prefer to keep the current links until the new docs are updated with more information. Right now the context page content is rather abstract and could use more concrete information on webview contexts (and links to driver-specific documentation as well). I would also suggest having (Appium-maintained) client examples like in the old docs.